### PR TITLE
Fixed two typos

### PR DESCRIPTION
**Pull request by AleksandrsBerdicevskis**
(description)

### CODE REVIEW

**Comment by bast** [comment]
Here I can ask questions, give suggestions ....

**Comment by AleksandrsBerdicevskis** [comment]
I answer and agree

File: random-text.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ back, with his ears depressed as if afraid, and kept looking me in the
 face, as if expecting me to pursue and shoot, for he had seen many a
 bear batttle in his day.
 
-Following the ridge, which made a gardual descent to the south, I came
+Following the ridge, which made a gradual descent to the south, I came
```
Comment (bast) [inline random-text.md:35]:
```suggestion
Following the ridge, which made a (some different change) descent to the south, I came
```

**Comment by bast** [comment]
Hello @lajanda.

**Pull request merged**